boot/zephyr/pm.yml

```diff
@@ -13,6 +13,15 @@ mcuboot_primary_app:
 mcuboot_primary:
   span: [mcuboot_pad, mcuboot_primary_app]
 
+#if defined(CONFIG_BOOT_SWAP_USING_MOVE)
+# This region is used to ensure that at least 1 page remains unused
+mcuboot_swap_area:
+  size: CONFIG_FLASH_LAYOUT_PAGE_SIZE
+  placement:
+    after: app
+  inside: mcuboot_primary_app
```

Comment:
Are you going to move it to the `mcuboot_primary` area (as we discussed) ?

Reply:
Its a while since we discussed this, what was the reason for placing it there instead of inside `mcuboot_primary_app` - it should have the same result no?